internal/activities/validate_sip_name.go

```diff
@@ -0,0 +1,38 @@
+package activities
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/artefactual-sdps/preprocessing-sfa/internal/sip"
+)
+
+const ValidateSIPNameName = "validate-transfer-name"
```

Comment:
Can the string be "validate-sip-name" too or is there a reason to keep using "transfer"?

Reply:
Haha, ValidateSIPNameName is pretty cute. 😆